hw03_samundra.py

```diff
@@ -0,0 +1,127 @@
+#################################################################################
+# Author: Samundra Adhikari
+# Username: hw03_samundra
+#
+# Assignment: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To familiarize functions, turtle graphics, computer color representation, and Git basics.
+# Google Doc Link: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.wh3weva4kufs
+#
+#################################################################################
+# Acknowledgements: Used ChatGPT to ask questions, and used textbook.
+#
+#
+#################################################################################
+
+import turtle
+
+
+# Draw the cat's head
```

Comment:
This should probably in be the docstring for `main()`, which is also missing

Reply:
I have corrected the code! Thank you